internal/venice-test-common/src/integrationtest/java/com/linkedin/venice/endToEnd/WriteComputeWithActiveActiveReplicationTest.java

```diff
@@ -836,19 +847,17 @@ public void testAAReplicationForPartialUpdateOnMapField() throws IOException {
     verifyFLMRecord(storeName, dc1RouterUrl, key2, regularFieldValue, arrayFieldValue, expectedMapFieldValue);
     verifyFLMRecord(storeName, dc0RouterUrl, key2, regularFieldValue, arrayFieldValue, expectedMapFieldValue);
 
-    // todo: Two partial puts with the same timestamp doesn't produce deterministic output
-    /*
     ub = new UpdateBuilderImpl(wcSchemaV1);
     mapFieldValue = new HashMap<>();
     mapFieldValue.put("eight", 88);
     mapFieldValue.put("seven", 77);
     ub.setNewFieldValue(MAP_FIELD, mapFieldValue);
     timestampedOp = new VeniceObjectWithTimestamp(ub.build(), 3);
     sendStreamingRecord(systemProducerMap.get(childDatacenters.get(0)), storeName, key2, timestampedOp);
-    expectedMapFieldValue.putAll(mapFieldValue);
```

Comment:
Can we document the tiebreaker used here?  And add one more case where PartialPut comes from colo-0 and it is processed before ParitalPut from colo-1.

Reply:
Will do. For test coverage, I think I added it in the unit test. I will try to see if it is easy to add it into integration test...